DevelopGitHub/GitHubExtension/GitHubExtension.Constant/RouteConstants.cs

```diff
@@ -9,35 +9,36 @@ namespace GitHubExtension.Constant
     public class RouteConstants
     {
         // route prefixes
-        public const string ApiRoles = "api/v1/roles";
-        public const string ApiAccount = "api/v1/account";
-        public const string ApiRepository = "api/v1/repository";
+        public const string ApiRoles = "api/roles";
+        public const string ApiAccount = "api/account";
+        public const string ApiRepository = "api/repos";
+        public const string ApiUser = "api/user";
 
         // static segments
         public const string User = "user";
-        public const string Repository = "repository";
```

Comment:
We will change routes from "repo/repos" to "repository" after creating the same file with routes' constants on FrontEnd part. 
I would prefer to create this file instead of fixing backend, but my colleage made other mistakes, so I fixed all problems on backend and temporaty renamed "repository" route.